jmclient/jmclient/wallet_utils.py

```diff
@@ -687,6 +692,12 @@ def wallet_generate_recover(method, walletspath,
     return True
 
 
+def wallet_change_passphrase(walletservice):
+    passphrase = cli_get_wallet_passphrase_check(changing_passphrase=True)
+    walletservice.wallet._storage.change_password(passphrase)
```

Comment:
This makes sense but we should probably just add a `wallet.BaseWallet.change_password()` method to avoid accessing the `_storage` member of the class (this would also avoid needing to add the `.wallet` part because we'd use the `getattr` access).